sycl/test-e2e/AddressSanitizer/common/statistics.cpp

```diff
@@ -0,0 +1,31 @@
+// REQUIRES: linux
+// RUN: %{build} %device_asan_flags -O2 -g -o %t
+// RUN: env SYCL_PREFER_UR=1 %{run} %t 2>&1 | FileCheck %s
+// RUN: env SYCL_PREFER_UR=1 UR_LAYER_ASAN_OPTIONS=print_stats:1 %{run} %t 2>&1 | FileCheck --check-prefixes CHECK-STATS %s
```

Comment:
if use env, then please use env for each environment variable.

Reply:
done